thangs_fetcher.py

```diff
@@ -118,7 +118,16 @@ def reset(self):
     def search(self, query):
         if self.searching:
             return False
-        self.query = urllib.parse.quote(query)
+        self.query = urllib.parse.quote(query, safe='')

```

Comment:
Any reason we don't do this right there in the line that opens the browser?  Seems like less chance of things like double-encoding and such there.